exampleplugins/5.2/SwagCustomProductBoxLayout/SwagCustomProductBoxLayout.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace SwagCustomProductBoxLayout;
+
+use Shopware\Components\Plugin;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Shopware-Plugin SwagCustomProductBoxLayout.
+ */
+class SwagCustomProductBoxLayout extends Plugin
+{
+
+    /**
+    * @param ContainerBuilder $container
+    */
+    public function build(ContainerBuilder $container)
+    {
+        $container->setParameter('swag_custom_product_box_layout.plugin_dir', $this->getPath());
```

Comment:
this is not needed since SW 5.2.xsomething. so you can remove the build method, as we have 5.5.3 now :wink: